### DISCUSSION TITLE
Requesting .txt File in Root Folder Example

### DISCUSSION THREAD

**Discussion by kaylaharrison** [CMSgov/hospital-price-transparency#1] [Q&A] [answered]
Hello, 

One of the accessibility requirements for the OPPS Final Rule that is due on 01/01/24 is including a .txt file in the root folder on the public website. Can an example be provided to ensure we're being compliant with CMS requirements?

**Comment by bill-citro** [comment]
I would like to second this ask. There is no clarity on the appropriate formatting of the text file.

**Comment by elissatucci** [comment]
We would also greatly appreciate further clarity regarding what to put for the hospital point of contact information.

**Comment by tomgladney** [comment]
My organization echos the request for a TXT file example for the price transparency root folder TXT file requirement due 1/1/24.  Thank you.

**Comment by mcrisanSP** [comment]
We're also in need of the .txt file generator and technical instructions. I am starting to get concerned with vendor turnaround time by 1/1/2024 if we don't get them instructions within the next week or two.

**Comment by shaselton-usds** [comment, marked as answer]
Hi @kaylaharrison @bill-citro @elissatucci @tomgladney @mcrisanSP -- thanks for your patience!

As finalized in CY2024 OPPS/ASC Final Rule, beginning January 1, 2024, each hospital must ensure that the public website it selects to host its machine-readable file (MRF) establishes and maintains, in the form and manner specified by CMS:

* A TXT file in the root folder that includes:
  * The hospital location name that corresponds to the machine-readable file;
  * The source page URL that hosts the machine-readable file;
  * A direct link to the machine-readable file (the machine-readable file URL); and
  * Hospital point of contact information.
* A link in the footer on its website, including but not limited to the homepage, that is labeled ‘‘Price Transparency’’ and links directly to the publicly available web page that hosts the link to the machine-readable file.

The purpose of these requirements is to facilitate automated access to hospital MRFs. Please refer to 45 CFR 180.50 (d)(6) and discussion at 88 FR 82111-82113.

Required information and technical specifications for the TXT file, as well as an example TXT file, are available on the [HPT Tools Webpage here](https://cmsgov.github.io/hpt-tool/txt-generator/).
